schema-engine/sql-schema-describer/src/postgres.rs

```diff
@@ -692,12 +717,15 @@ impl<'a> SqlSchemaDescriber<'a> {
                 BitFlags::empty()
             };
 
+            let constraints_key = (namespace_cloned, table_name_cloned);
```

Comment:
Why not just `.clone()` them here if this is the only place we're using those newly introduced variables?

Reply:
Yeah, we don't need `_cloned` variables, just call `.clone()` directly where you need it.